blog/2024-08-14-headlamp/index.mdx

```diff
@@ -0,0 +1,86 @@
+---
+slug: headlamp
+title: "Repost: Streamlining Kubernetes Cost Management with the New OpenCost Plugin for Headlamp"
+authors: [team]
+tags: [opencost, headlamp, kubernetes, ui, integrations]
+---
+<head>
+    <link rel="canonical" href="https://headlamp.dev/blog/2024/08/08/opencost-plugin"/>
```

Comment:
Anything else we need to do to properly support a repost? I'm less familiar here.

Reply:
I am actually not sure. I just copied one of the reposts you had and changed accordingly.

Reply:
I'm not an expert but looks fine...

Reply:
cc @joe-terrible @dzeiskubecost